### ISSUE TITLE
AEM Environment Indicator Not Sticky to Header | AEM Cloud

### ISSUE THREAD

**Issue by vivekdhiman** [Adobe-Consulting-Services/acs-aem-commons#3020]
Issue is on the AEM cloud where user logged in via adobe id (SSO)

### Expected Behavior

As per the implementation, Indicator should be stick with the top nav.

### Actual Behavior

If user log into the cloud environment via adobe id. User see the indicator not stock with the top nav.

### Steps to Reproduce

Logged with adobe id and authenticate with SSO.
User will see Landing page of AME Sites and see the env. indicator is not sticky on the top nav.

We observed the header is spectrum-Shell spectrum--darkest header which was earlier  coral-shell-header in AEM Cloud

### Links

_Links to related assets, e.g. content packages containing test components_
![ss](https://user-images.githubusercontent.com/7066652/211878778-ce28a96c-4595-433e-a243-b5ca86f5f968.png)

**Comment by davidjgonzalez** [comment]
@vivekdhiman what is your OSGi configuration for injecting the indicator? Is the problem the header is changing classes? Ultimately you should be able to use the css-override OSGi config to position your indicator where you want it. You can alos use that `inner-html` to include JS that can perform fancier/checks and manipulations.

Also, it looks like you are NOT using Unified Shell, is that right?

**Comment by vivekdhiman** [comment]
Hi @davidjgonzalez , We are able to fix this with custom JS and CSS. I believe all the cloud environments where user login with SSO this issue exists. Below is the config we used earlier
```
{
    "browser-title-prefix":"Author",
    "css-color":"#FF7F50",
    "css-override":"#acs-commons-env-indicator {                     background-color: #800080;                     color: #FFF;                     position: fixed;                     top: 0;                     left: 50%;                     width: 126px;                     margin-left: -63px;                     height: 30px;                     font: bold 18px/29px sans-serif;                     text-align: center;                     border: solid 1px black;                     border-top-width: 0;                     z-index: 100000000000000;     }",
    "inner-html": "STAGE<script>document.getElementById('acs-commons-env-indicator').onmouseover = function() { if(this.style.left === '50%') { this.style.left = '25%';} else {this.style.left = '50%'; }};</script>"
}
```

**Comment by davidjgonzalez** [comment]
This should be fixed in 6.0.0. 
IIUC the problem was caused by Unified Shell (the new white top nav bar in AEM CS). Unified shell brings in AEM CS as an iframe under it - so some of the JS to inject the indicator didnt work quite right.

It's worth noting that AEM CS now indicates prod/stage/dev/rde OOTB in a little pill in the top left, so not sure how valuable this feature is in AEM CS now.

If you can re-open is youre still seeing strange behavior, or if i misunderstood what you were reporting.
